pandas/tests/extension/base/methods.py

```diff
@@ -240,23 +240,26 @@ def test_factorize_empty(self, data):
         tm.assert_numpy_array_equal(codes, expected_codes)
         self.assert_extension_array_equal(uniques, expected_uniques)
 
-    def test_fillna_copy_frame(self, data_missing):
+    def test_fillna_copy_frame(self, data_missing, using_copy_on_write):
```

Comment:
If we need to use the fixture here (in the /extension) tests, then ideally we redefine it in `extension/conftest.py`, to make it easier for downstream authors. 

It might become unavoidable to use it at some point, but for this specific case we could maybe also verify the "copy" by modifying the result and ensuring the parent didn't update. In the end, that's the behaviour we care about for users, the values being identical is more an implementation detail.

Reply:
I restructured the tests, but we need using_copy_on_write for the sparse test, so I copied it to conftest.py in extension

Reply:
> but we need using_copy_on_write for the sparse test, so I copied it to conftest.py in extension

Sidenote: if it's only for the sparse test override, and not in the base class definitions, then we don't need it in the conftest.py in extension/ (because it's only needed if it is relevant for external packages subclassing those base test classes as well)